workers/tests/integrationtests/test_api.py

```diff
@@ -21,13 +21,18 @@ def setUp(self):
         self.result_schema = get_result_schema()
 
     def test_get_available_regions(self):
-        url = "/regions"
+        url = "/regions?asGeoJSON=true"
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
         response_content = json.loads(response.content)
         self.assertTrue(self.general_schema.is_valid(response_content))
         result = geojson.loads(json.dumps(response_content))
         self.assertTrue(result.is_valid)
+        url = "/regions?asGeoJSON=false"
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        response_content = json.loads(response.content)
+        self.assertTrue(self.general_schema.is_valid(response_content))
```

Comment:
Is it possible to request the enpoint `/regions` without any parameter?

I would add a test for this as well.